types/chai/index.d.ts

```diff
@@ -422,110 +422,101 @@ declare namespace Chai {
         /**
          * Throws a failure.
          *
-         * @type T   Type of the objects.
          * @param actual   Actual value.
          * @param expected   Potential expected value.
          * @param message    Message to display on error.
          * @param operator   Comparison operator, if not strict equality.
          * @remarks Node.js assert module-compatible.
          */
-        fail<T>(actual: T, expected: T, message?: string, operator?: Operator): never;
+        fail(actual: unknown, expected: unknown, message?: string, operator?: Operator): never;
```

Comment:
The old form would report an error at design-time for `fail("1", 2)`, but the new form doesn't.  Intentional?

Silly question: why would a `fail` function take expected and actual values?

Reply:
Doesn't match any of the testing libraries I've used, but sounds intentional.